cameronRomo/my-strain-book
# Feature/enable api call

### What’s this PR do?

* Adds an api call to strains database
* Adds conditional rendering to show loading bar while resolving the promise
*  Enables `NavBar` to use routes for switching between components/pages

### Where should the reviewer start?

* `src/App/App.js` line 14
* `src/Card/Card.js` line 1
* `src/Experiences/Experiences.js` line 4
* `src/NavBar/NavBar.js` line 2
* `src/Strains/Strains.js` line 1
* `src/apiCalls.js` line 1

### How should this be manually tested?

* Pull down, on strains note loading message, open up drop-down and see all strains, click through nav links at the top to see different pages

### Any background context you want to provide?

* An API call was needed to grab all the strain data from the API and display it. While that is happening it seemed best to let the user know something was happening behind the scenes. Router was the way I enabled the user to effectively switch between components/pages.

### Questions:
* I don't know why .env is still showing up. It's definitely in `.gitignore`. Any clues??
